### ISSUE TITLE
Running test "@nrwl/jest" on version 11.2.10 via npm, returns Exit code 0, even if has failed tests

### ISSUE THREAD

**Issue by besgit** [nrwl/nx#4711]
Running test "@nrwl/jest" on version 11.2.10 via npm, returns Exit code 0, even if has failed tests. In version 11.1.5 it works normaly. We have problems running test with jenkins, test plan always green.

Is this normal behavior for @nrwl/jest?

**Comment by besgit** [comment]
Resolved by updating angular-cli to version 11.1.3

**Comment by besgit** [comment]
Problem stay actual

**Comment by AdamJessop** [comment]
+1 facing this issue also, believe this is a bug rather than a discussion.

**Comment by artaommahe** [comment]
Same with `nrwl/jest@11.2.6` Worked well with 10.x version

**Comment by MatthewS2077** [comment]
+1 jest exits with Exit code 0 when test fails... sort of makes Jest useless if our build scripts can't detect test failure 🤷‍♂️

**Comment by MatthewS2077** [comment]
Seems to be fixed in 11.2.12

**Comment by artaommahe** [comment]
> Seems to be fixed in nrwl/jest 11.2.12

still exit code 0 with failed test and `nrwl/jest 11.3.0`

```
13:59:53  Test Suites: 1 failed, 66 passed, 67 total
13:59:53  Tests:       1 failed, 701 passed, 702 total
13:59:53  Snapshots:   0 total
13:59:53  Time:        37.077 s
13:59:53  Ran all test suites.
13:59:53   
13:59:53  [shell] out> Done in 44.30s. 
```

**Comment by FrozenPandaz** [comment]
Sorry for the delay everyone.

I found the issue :tada: 

https://github.com/nrwl/nx/commit/f206925abf43c32c83b566a88677555dadb0cff7 fixes the issue and has been released with `11.3.2`.
